### ISSUE TITLE
Log to GA Traceback

### ISSUE THREAD

**Issue by jag3773** [unfoldingWord-dev/tools#19]
Here is a traceback that was causing the same request to be reposted several times, at least from the standpoint of the screen output.

```
2015-04-06 18:40:54,214 - ERROR - Bad HTTP request,  502: Bad Gateway
2015-04-06 20:09:55,459 - ERROR - Traceback (most recent call last):
  File "log_to_ga.py", line 131, in <module>
    process_this_line(line)
  File "log_to_ga.py", line 93, in process_this_line
    connection.request('POST', '/collect', '&'.join(payload))
  File "/usr/lib64/python2.7/httplib.py", line 973, in request
    self._send_request(method, url, body, headers)
  File "/usr/lib64/python2.7/httplib.py", line 1007, in _send_request
    self.endheaders(body)
  File "/usr/lib64/python2.7/httplib.py", line 969, in endheaders
    self._send_output(message_body)
  File "/usr/lib64/python2.7/httplib.py", line 829, in _send_output
    self.send(msg)
  File "/usr/lib64/python2.7/httplib.py", line 791, in send
    self.connect()
  File "/usr/lib64/python2.7/httplib.py", line 772, in connect
    self.timeout, self.source_address)
  File "/usr/lib64/python2.7/socket.py", line 553, in create_connection
    for res in getaddrinfo(host, port, 0, SOCK_STREAM):
gaierror: [Errno -2] Name or service not known
```

**Comment by phillip-hopper** [comment]
@jag3773 , does this help: http://stackoverflow.com/questions/4673166/python-httplib-name-or-service-not-known

**Comment by dsm-git** [comment]
Phil, can you wrap the problem lines in a try/except statement?

**Comment by phillip-hopper** [comment]
They are already inside a try/except statement, that's why it logged the full stack trace.  It was sending the same line multiple times because it was set to send the last line from the log as soon as the script started, so each time it was started it sent the same line (apparently the site didn't get much traffic).  I disabled that feature, so it will only send newly added lines.

I think the actual problem might be http vs. https.  I set it to explicitly use https now, so hopefully that will fix the problem.
